.github/workflows/monitor-components.yml

```diff
@@ -12,12 +12,11 @@ on:
 
 env:
   CHARACTER_LIMIT: 5000
-  MAX_AGE: 48h
+  MAX_AGE: 7d
 
 jobs:
   job:
     runs-on: ubuntu-latest
-    environment: monitor-components
```

Comment:
I believe I added this to allow the workflow to open issues, using an environment secret.

A better way (that I was unaware of, at the time I created that environment) would probably be something along these lines:

```yaml
permissions:
  issues: write
```

Reply:
'